styles/toast.css

```diff
@@ -0,0 +1,43 @@
+.toast-container{
+    background-color: var(--text-color-black);
+    color: var(--body-background-color);
+    display: flex;
+    flex-direction: row;
+    padding: 1rem;
+    margin: 1rem;
+    width: fit-content;
+    border-radius: 0.6rem;
+}
+
+.toast-text{
+    margin-right: 1rem;
+    font-size: bold;
+}
+
+.retry{
+    cursor: pointer;
+    margin-right: 1rem;
+}
+
+.retry:hover{
+    color: var(--hover-color);
+}
+
+.toast-icon{
+    cursor: pointer;
```

Comment:
make cursor utility class and use whereever required

Reply:
ok